package.json

```diff
@@ -45,13 +45,18 @@
     "express-async-handler": "^1.1.4",
     "express-response-errors": "^1.0.4",
     "fork-ts-checker-webpack-plugin": "^1.5.1",
+    "immer": "^3.2.0",
+    "isomorphic-unfetch": "^3.0.0",
```

Comment:
We have `isomorphic-fetch` in PR #140 , can you explain why we should use `unfetch` instead of `fetch`?

Reply:
The main reason to use `isomorphic-unfetch` is for it's small bundle size. But in my opinion, if we are ok to use bigger bundled libs, `axios` would be a preferred option, in my opinion.

Reply:
I think we should discuss this further, but I think that bundle size isnt that big of a problem for now

Reply:
I have updated the PR to use `node-fetch` in server and `github/fetch` in browser, as both these packages are actively maintained and quite stable.

This change is on behalf of the discussion @Sonicrida and I had, with approval of few other channel members.